scripts/src/java/org/oppia/android/scripts/ci/BUILD.bazel

```diff
@@ -31,3 +31,34 @@ kt_jvm_library(
         "//scripts/src/java/org/oppia/android/scripts/proto:affected_tests_java_proto",
     ],
 )
+
+kt_jvm_library(
+    name = "compute_changed_files_lib",
+    testonly = True,
+    srcs = [
+        "ComputeChangedFiles.kt",
+    ],
+    visibility = ["//scripts:oppia_script_binary_visibility"],
+    deps = [
+        "//scripts/src/java/org/oppia/android/scripts/common:git_client",
+        "//scripts/src/java/org/oppia/android/scripts/common:proto_string_encoder",
+        "//scripts/src/java/org/oppia/android/scripts/common:repository_file",
+        "//scripts/src/java/org/oppia/android/scripts/proto:changed_files_java_proto",
+        "//scripts/src/java/org/oppia/android/scripts/proto:script_exemptions_java_proto",
```

Comment:
### Todo:
To remove script exemption deps with removal of test file exemptions

Reply:
Removed it for the compute_changed_files_lib, yet to be decided for retrieve_changed_files. - https://github.com/oppia/oppia-android/pull/5465#discussion_r1707454378

Reply:
Is this addressed now (i.e. can this conversation thread now be resolved)?

Reply:
Yeah, removed the dependencies for compute changed files while leaving it to stay as needed for retrieve changed files. 
Will resolve this comment.